src/http/controllers/leads/route.ts

```diff
@@ -15,4 +17,8 @@ export async function leadsRoute(app: FastifyInstance) {
   app.get('/lead/:id', getLead)
 
   app.put('/lead/:id', Update)
+
+  app.patch('/lead/:id', UpdateArchived)
```

Comment:
acho q fica melhor assim
```suggestion
  app.patch('/lead/arquived/:id', UpdateArchived)
```